internal/utils/test/fixtures.go

```diff
@@ -0,0 +1,42 @@
+package test
+
+import (
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/bluecmd/fortigate_exporter/internal/utils/files"
+)
+
+func GetFixturePath(fixtureFileName string) (string, error) {
+	callerDir, err := files.GetCallerDir(2)
```

Comment:
Don't use magic numbers like "2". It seems like we use it a lot in this code here, so you probably want to define an alias or helper function.

Reply:
`2` means in this context simply the function above me. 

```
aMethod --> GetFixturePath --> GetCallerDir
    2               1                0
```

What would you name the constant? I have no really nice constant name right now. 

The other alternative would a default value: 
```golang 
func GetRelativeFixturePathPanic(fixtureFileName string, additionalParameters ...interface{}) string {
	stackTraceStep := 2
	if len(additionalParameters) >= 1{
		stackTraceStep = additionalParameters[1].(int)
	}
	callerDir, err := files.GetCallerDir(stackTraceStep)
	if err != nil {
		log.Panicf("failed to get caller stack %v", err)
	}
	workDir, err := os.Getwd()
	if err != nil {
		log.Panicf("failed to get workdir %v", err)
	}
	relDir := strings.TrimPrefix(callerDir, workDir)
	fixturePath := filepath.Join(relDir, "testdata", "fixtures", fixtureFileName)
	return fixturePath
}

```

Reply:
I would probably write it like this:

```go

type Depth int

const (
        ONE_UP Base = 2
        TWO_UP Base = 3
        ...
)

func MustFixturePath(depth Depth) string {
	_, filename, _, ok := runtime.Caller(int(depth) /* stack depth to descend */)
	if !ok {
                panic(...)
	}
	workDir, err := os.Getwd()
	if err != nil {
		log.Panicf("failed to get workdir %v", err)
	}
	relDir := strings.TrimPrefix(path.Dir(filename), workDir)
	return filepath.Join(relDir, "testdata", "fixtures")
}
```

Though - do we actually need to construct these? Can't we

Reply:
As long you run `go test ./...` or you run test in the directory, we are fine removing then.